app_allocator/classes/option_spec.py

```diff
@@ -0,0 +1,4 @@
+class OptionSpec(object):
```

Comment:
Not a blocker, but this sort of minimal class seems like the sort of thing that the `namedtuple` was created for.

Reply:
This might change soon, so I'm not going to change it, but point taken.  May change later.

Reply:
👍